API.md

```diff
@@ -30,10 +30,15 @@ Reauthentication can be performed at anytime and will result in a brand new toke
 
 ### Scheme: OpenID Connect (OIDC)
 
-For authenticating with OpenID Connect, there are two methods:
+For authenticating with OpenID Connect, there are three methods, two of which are run as clients:
 
-- Resource Owner Password Credentials Grant
-- Authorization Code Grant
+- Client-Based
+  - Resource Owner Password Credentials Grant
+  - Authorization Code Grant
+- Protected Resource
+
+To run a client-based scheme, you must set `'mode'` in the `openid_connect` stanza to `'client'`.
+Otherwise, to run the HTTP API as an OAuth protected resource, set `'mode'` to `'protected_resource'`.
```

Comment:
_This is motivated by my previous comment._

What if each section had a line that stated something like ...
```
To enable this auth flow, under the `openid_connect` stanza, set `mode` to `client`.
```

Then everything the admin needs for understanding how to make it work is right there within each section.

These lines would be removed, of course.

Thoughts?

Reply:
Can do.

Reply:
Done.